src/nanopolish_call_methylation.cpp

```diff
@@ -199,12 +202,11 @@ void calculate_methylation_for_read(const OutputHandles& handles,
         std::vector<int> cpg_sites;
         assert(ref_seq.size() != 0);
         for(size_t i = 0; i < ref_seq.size() - 1; ++i) {
-            if(ref_seq[i] == 'C' && ref_seq[i+1] == 'G') {
+            if(mtest_alphabet->is_motif_match(ref_seq, i))
                 cpg_sites.push_back(i);
```

Comment:
can you rename this variable to something like motif_sites as they are no longer just cpg